src/Engine/Renderer/RenderTechnique/ShaderProgram.cpp

```diff
@@ -186,6 +186,24 @@ void ShaderProgram::load( const ShaderConfiguration& shaderConfig ) {
     }
 
     link();
+
+    int texUnit = 0;
+    auto total = m_program->get( GL_ACTIVE_UNIFORMS );
+    textureUnits.clear();
+
+    for ( int i = 0; i < total; ++i )
```

Comment:
why not declaring ``i`` as ``GLuint`` instead of declaring it as ``int`` and always casting below ?